packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1771,7 +1784,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
   };
 
   private _onOptionMouseEnter(index: number): void {
-    if (!this._isScrollIdle) {
+    if (!this._isScrollIdle || !this._gotMouseMove) {
```

Comment:
>(!this._isScrollIdle || !this._gotMouseMove) [](start = 7, length = 44)

perhaps move this logic into a function that you can share with _onOptionMouseLeave ? that way also you an name it which makes the code easier to read, something like shouldUpdatePendingValue or something

Reply:
pulled it into a shouldIgnoreMouseEvent function